backend/satellite_tools/reposync.py

```diff
@@ -506,15 +507,17 @@ def sync(self, update_repodata=True):
         repo_checksum_type = 'sha1'
         start_time = datetime.now()
         for data in self.urls:
-            data['source_url'] = self.set_repo_credentials(data)
+            data['source_url_auth'] = self.set_repo_credentials(data)
+            data['source_url'] = [u["url"] for u in data['source_url_auth']]
```

Comment:
It's not in use in this method. The field `source_url` is used as input for the reposync processing. So I decided to keep the same logic for this field just in case it was use in outside in any other context I would not be aware of.

Reply:
Just wondering here if there is any particular reason to introduce this new `data['source_url_auth']` field here and not just keep using `data['source_url_auth']`?